### PR TITLE
Add Lord Of Magna: Maiden Heaven

### PR DESCRIPTION
**Pull request by Azursoul**
Adds Lord Of Magna: Maiden Heaven.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by Azursoul** [comment]
resized box art
lord of magna_boxart

**Comment by Azursoul** [comment]
do i need to show the frame rate stats under the screen?

**Comment by CodingKoopa** [comment]
> resized box art

Please update [your branch](https://github.com/Azursoul/citra-games-wiki/tree/pr-builder-lord-of-magna-maiden-heaven-90) with it.

> do i need to show the frame rate stats under the screen? since by respecting the size asked i needed to cut it off

No, that shouldn't be included.

File: games/lord-of-magna-maiden-heaven/game.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "Lord Of Magna: Maiden Heaven"
+description = "by Marvelous, genre: RPG\n\nMagna puts you in the shoes of a humble innkeeper (whose name can be customized at the start), who made a promise to his father to always keep his doors open and be the best proprietor he can be. One day on a standard trip to a cave to gather crystals (a precious resource in this world), he encounters a group of monsters. Fearing death, he retreats to a corner hosting a giant crystal, and summons a magical Spirit Girl named Charlotte (an \"Artemis\")"
```
[the remark this answers is not in the document]
Comment (CodingKoopa) [reply games/lord-of-magna-maiden-heaven/game.dat]: If this description is used, the Japanese release should be added, too.

**Comment by Azursoul** [comment]
the unused space at the top of above-60-fps.png is due the the movie's size and turning the white side of the touch screen black (should be black but is white for me) 
movie screen

**Comment by Azursoul** [comment]
update on the box art as been done, and added the Japanese version as well

**Comment by Leo626** [comment]
I don't see where you are pushing your changes. I don't see the commits. Changes should be done in your own branch, inside your repository [here](https://github.com/Azursoul/citra-games-wiki/tree/pr-builder-lord-of-magna-maiden-heaven-90/games/lord-of-magna-maiden-heaven). 

The screenshot background (black bars in bottom screen) should be black. Ensure you the background settings have been set correctly. See the [contributing guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#game-screenshots) on this. There should be at least 3 screenshots.

**Comment by Azursoul** [comment]
trying to take screenshot in battle but the game works at speed: 14%/100%

**Comment by Leo626** [comment]
There should only be one box art (boxart.png) and it should be USA region one.

File: games/lord-of-magna-maiden-heaven/game.dat [hunk 2]
```diff
@@ -0,0 +1,13 @@
+title = "Lord Of Magna: Maiden Heaven"
+description = "by Marvelous, genre: RPG, Lord of Magna: Maiden Heaven, known as "Forbidden Magna" in Japan, is a tactical role-playing life simulation published by Marvelous. with localization by XSEED for the West. It shares several similarities with the Rune Factory series.
```
Comment (Leo626) [inline games/lord-of-magna-maiden-heaven/game.dat]: Don't start the sentence with `by Marvelous`. Use the description I gave you [before](https://github.com/citra-emu/citra-games-wiki/pull/392#discussion_r204208717).

File: games/lord-of-magna-maiden-heaven/game.dat [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+needs_system_files = true
+needs_shared_font = true
+game_type = "eShop/Application"
```
Comment (Leo626) [inline games/lord-of-magna-maiden-heaven/game.dat]: Again, this line is not needed and is not even the correct use.

File: games/lord-of-magna-maiden-heaven/game.dat [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+[[releases]]
+title = "0004000000164300"
+region = "usa"
+release_date = "2015-06-02"
+title = "0004000000165500
```
Comment (Leo626) [inline games/lord-of-magna-maiden-heaven/game.dat]: The should be `[[releases]]` above this and the releases should be separated with a new line.

**Comment by Azursoul** [comment]
is the error in this?

title = "Lord Of Magna: Maiden Heaven"
description = "\nLord of Magna: Maiden Heaven, known as "Forbidden Magna" in Japan, is a tactical role-playing life simulation published by Marvelous. with localization by XSEED for the West. It shares several similarities with the Rune Factory series."
needs_system_files = true
needs_shared_font = true

[[releases]]
title = "0004000000164300"
region = "usa"
release_date = "2015-06-02"

title = "0004000000165500
region = "eur"
release_date = "2015-06-04"

File: games/lord-of-magna-maiden-heaven/game.dat [hunk 5]
```diff
@@ -0,0 +1,15 @@
+title = "Lord Of Magna: Maiden Heaven"
+description = "Lord of Magna: Maiden Heaven, known as \"Forbidden Magna\" in Japan, is a tactical role-playing life simulation published by Marvelous. with localization by XSEED for the West. It shares several similarities with the Rune Factory series."
+needs_system_files = true
+needs_shared_font = true
+game_type = "Application"
```
Comment (Leo626) [inline games/lord-of-magna-maiden-heaven/game.dat]: Let me explain again. This line (`game_type = "Application"`) is not needed and should be deleted. `Application` is not even a correct game type.

**Comment by Azursoul** [comment]
finally

File: games/lord-of-magna-maiden-heaven/game.dat [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+[[releases]]
+title = "0004000000164300"
+region = "usa"
+release_date = "2015-06-02"
+
+[[releases]]
+title = "0004000000165500"
+region = "eur"
+
```
Comment (Leo626) [inline games/lord-of-magna-maiden-heaven/game.dat]: Unnecessary new line.

**Comment by Leo626** [comment]
Give the screenshots some descriptive names like `cutscene.png` or `gameplay.png`. Also there is a `jpn` release. Title ID is `0004000000120900`.

**Comment by Azursoul** [comment]
still don't have the battle screen yet (better to take then from citra?)

**Comment by Azursoul** [comment]
at the rate the game's goes, i need a good 30 min to get the 2 last screenshot

**Comment by Azursoul** [comment]
took me much longer then expected

**Comment by Leo626** [comment]
Still need to address this https://github.com/citra-emu/citra-games-wiki/pull/392/files#r204222604

**Comment by Leo626** [comment]
Nope, separate the releases like [this](https://github.com/citra-emu/citra-games-wiki/blob/master/games/monster-hunter-stories/game.dat).

**Comment by Azursoul** [comment]
is there anything missing or a step i missed?

**Pull request merged**